### PR TITLE
#5179 Article landing page should adjust heading levels hierachy

### PR DESCRIPTION
**Pull request by thinkbulecount2**
Additional comments by @israelcefrin and @NateWr on this issue: https://github.com/pkp/pkp-lib/issues/5179

### CODE REVIEW

File: templates/frontend/objects/article_details.tpl [hunk 1]
```diff
@@ -259,6 +260,7 @@
 			{/if}
 			{if $supplementaryGalleys}
 				<div class="item galleys">
+					<h3 class="pkp_screen_reader">Additional Files</h3>
```
Comment (NateWr) [inline templates/frontend/objects/article_details.tpl:263]: This (and "Downloads" above) need to be translatable strings that use `{translate key="..."}`.

Comment (thinkbulecount2) [reply templates/frontend/objects/article_details.tpl:263]: I'm not sure about these namings, do you think `submission.layout.galley.header` and `submission.layout.galley.additionalHeader` work?

Comment (NateWr) [reply templates/frontend/objects/article_details.tpl:263]: I like to keep it simple. Try `submission.downloads` and `submission.additionalFiles`.

Comment (thinkbulecount2) [reply templates/frontend/objects/article_details.tpl]: Try now? @NateWr

**Comment by NateWr** [comment]
Replaced by https://github.com/pkp/ojs/pull/2752

**Pull request closed without merging**